python/tests/integration_tests/test_terra_util.py

```diff
@@ -117,61 +118,64 @@ def test_update_multiple_users_acl():
             assert invite["canShare"] is True
 
 
-def test_create_workspace_attributes_ingest_dict():
+def test_create_workspace_attributes_ingest_dict() -> None:
     res = terra_workspace.create_workspace_attributes_ingest_dict()
     assert res == [{"attribute": "library:dulvn", "value": "1"}]
 
 
-def test_upload_metadata_to_workspace_table():
+def test_upload_metadata_to_workspace_table() -> None:
     current_dir = os.path.dirname(__file__)
     file_path = os.path.join(current_dir, "sample.tsv")
     res = terra_workspace.upload_metadata_to_workspace_table(entities_tsv=file_path)
     assert res == "sample"
 
 
-def test_get_workspace_workflows():
+def test_get_workspace_workflows() -> None:
     res = terra_workspace.get_workspace_workflows()
     assert res == []
 
 
-def test_import_workflow():
-    workflow = "ExportDataFromSnapshotToBucket"
-    workflow_config = getattr(WorkflowConfigs(), workflow)(billing_project=INTEGRATION_TEST_TERRA_BILLING_PROJECT)
-    status_code = terra_workspace.import_workflow(workflow_dict=workflow_config)
+def test_import_workflow() -> None:
+    workflow_name = "ExportDataFromSnapshotToBucket"
+    status_code = WorkflowConfigs(
+        workflow_name=workflow_name,
+        billing_project=INTEGRATION_TEST_TERRA_BILLING_PROJECT,
+        terra_workspace_util=terra_workspace
+    ).import_workflow()
```

Comment:
If the `import_workflow` method doesn't return anything and we can't make an assertion, we should either update the method to return a status code, or remove this test (since it's not actually testing anything)

Reply:
This should be returning the return code